translations/monero-core_ro.ts

```diff
@@ -1759,34 +1767,35 @@ For the case of outgoing payments, you can get a &apos;Spend Proof&apos; that pr
         <location filename="../pages/TxKey.qml" line="159"/>
         <location filename="../pages/TxKey.qml" line="271"/>
         <source>Message</source>
-        <translation type="unfinished"></translation>
+        <translation>Mesaj</translation>
     </message>
     <message>
         <location filename="../pages/TxKey.qml" line="166"/>
         <location filename="../pages/TxKey.qml" line="278"/>
         <source>Optional message against which the signature is signed</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Mesaj opțional atașat semnăturii</translation>
```

Comment:
If the translation is correct please remove `type="unfinished"`

Reply:
**Done**. There are still 14 unfinished: for 11 of them I'm in doubt and 3 of them I cannot translate.

To recap:
- Inside `<name>LeftPanel</name>`:
I don't know what these mean: R, K, B, H, D, M, Y, E S, I

- Inside `<name>AddressBook</name>`, this one seems dubious in English:
`Payment ID. A unique user name used in the address book. It is not a transfer of information sent during the transfer.`

- And I was unable to translate:
`Successfully rescanned spent outputs`
`Seep unmixable`
`No unmixable outputs to sweep`